Modules/_typingmodule.c

```diff
@@ -1,6 +1,9 @@
 /* typing accelerator C extension: _typing module. */
 
+#define Py_BUILD_CORE
```

Comment:
Looks like there's some compiler warnings on this line on Windows

Reply:
https://github.com/python/cpython/actions/runs/4836719369/jobs/8620095794?pr=103764#step:4:231:

> D:\a\cpython\cpython\Modules\_typingmodule.c(3,1): message : 'Py_BUILD_CORE' previously declared on the command line [D:\a\cpython\cpython\PCbuild\pythoncore.vcxproj]